playroom/snippets.tsx

```diff
@@ -2833,11 +2833,72 @@ const ProgressBlockSnippets = [
     },
 ];
 
+const timerSnippets: Array<Snippet> = [
+    {
+        group: 'Timer',
+        name: 'No Label',
+        code: `
+          <Timer
+            endTimestamp={Date.now() + 1000 * 60 * 60 * 24}
+            minTimeUnit="seconds"
+            maxTimeUnit="days"
+          />
+        `,
+    },
+    {
+        group: 'Timer',
+        name: 'Short Label',
+        code: `
+          <Timer
+            endTimestamp={Date.now() + 1000 * 60 * 60 * 24}
+            minTimeUnit="seconds"
+            maxTimeUnit="days"
+            labelType="short"
```

Comment:
does the `Timer` have `labelType`? I only see it in the props of `TextTimer`

Reply:
Old snippets. Changed